targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/TARGET_MUSCA_A1_S/device/drivers/mpu_armv8m_drv.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2017-2018, Arm Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Dont forget to document  this BSD-3-Clause (https://os.mbed.com/docs/mbed-os/v5.11/contributing/license.html - Using different license).